index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Clicker+Script&family=Poppins:wght@400;500;600;700&display=swap');
+    </style>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="index.css">
+    <title>Protfolio Web</title>
+</head>
+
+<body>
+
+    <header class="navbar">
+        <a class="logo" href="#">My Logo</a>
+        <nav>
+        <img class="hamburg" src="assets/hamburg.svg" alt="hamburg">
+    </nav>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you correctly align your tags to make your code base more readable. 
```HTML
<!-- bad -->
        <nav>
        <img class="hamburg" src="assets/hamburg.svg" alt="hamburg">
    </nav>

    <!-- good -->
    <nav>
      <img class="hamburg" src="assets/hamburg.svg" alt="hamburg">
    </nav>
```